# Transgenerational Programming of Methylation and Gene Expression in the Prefrontal Cortex (PFC) by Glucocorticoids

# Introduction: 
Prenatal exposure to excess glucocorticoids increases risk for neurodevelopmental disorders. Previous research has shown that prenatal glucocorticoid exposure results in offspring that are hyperactive in a stressful environment, and that this behavioral phenotype that is transmitted over multiple generations. The PFC is implicated in programming stress responsivity and behavior, and has previously been shown to be affected by prenatal glucocorticoids. Understanding the relationship between gene expression and phenotype provides greater insight into the molecular mechanisms which are being affected by prenatal synthetic glucocorticoid (sGC) exposure, and the inheritance of these mechanisms. Here, we hypothesized that prenatal sGC exposure programs gene transcription and methylation in the PFC across three generations following paternal transmission. Furthermore, we hypothesized that predictive modelling can be used to understand the relationship between gene expression and behavior. 

# Methods: 
Pregnant guinea pigs received 3 courses of betamethasone (Beta; 1mg/kg) or saline in late gestation. F1 and F2 male offspring were mated with non-experimental females to generate F2 and F3 offspring. Total locomotor activity in open-field (OFA) was measured in female offspring on postnatal day 24 and brains collected at day 40. The PFC was micro-punched from F1(C; n=4, Beta; n=4), F2(C; n=4, Beta; n=4), and F3(C; n=4, Beta; n=4) and RNA/DNA extracted. RNA was submitted for RNA-seq and data analyzed using standard bioinformatics. Hierarchal clustering was carried out on the OFA scores and normalized expression profiles of genes that were significantly differentially expressed across all 3 generations of animals. Multiple regression integrated gene expression profiles to predict OFA, and linear regression determined the correlation of predicted and observed OFA. The model was validated using leave-one-out cross validation, followed by the application of the model to a novel qRT-PCR data set. The model was trained on qRT-PCR data from samples that had been originally included in the RNA sequencing, and then tested on qRT-PCR data from additional samples (n=12). Methylation analyses were performed using bisulfite converted DNA paired with pyrosequencing of proximal transcription start site regions of target genes. In silico transcription factor binding analyses were undertaken using SABiosciencesâ€™ DECODE tool. 

# Results: 
In F1, 1148 genes were significantly (P<0.001, FDR<0.05) differentially expressed following prenatal Beta. Of these, 442 genes were significantly up-regulated, and 706 genes down-regulated. In F2 offspring, 432 genes were significantly (P<0.001, FDR<0.05) differentially expressed between Veh and Beta, with 255 genes up-regulated, and 177 genes down-regulated. In F3, 438 genes were significantly (P<0.001, FDR<0.05) differentially expressed following prenatal Beta. Of these, 258 genes were significantly up-regulated, and 180 genes down-regulated. 22 genes were significantly differentially expressed across all three generations. 16 out of the 22 significantly differentially expressed genes contained MEF2C binding site within 30kb of transcription start site. Mef2c expression was significantly (P<0.001, FDR<0.05) upregulated in F1 offspring, significantly (P<0.001, FDR<0.05) down regulated in F2 offspring, and trending towards significantly (P=0.009, FDR=0.15) increased in F3 animals. Methylation analyses revealed significant (p < 0.05) changes in methylation that significantly (p<0.05) correlated in gene expression changes across the generations. F2 animals cluster separately and distinctly from F1 and F3 animals. Multivariate linear regression demonstrated that OFA is significantly associated with expression of B3Galt2 (glycosyltransferase), Garnl3 (G-protein modulator), Nrip3 (transcription cofactor) & Gabra3 (GABA receptor) in F1 and F3 animal RNA-seq data (R2=0.72, p=0.0008). Cross validation revealed a significant correlation between predicted and observed OFA (R2=0.46, p=0.0003). The model replicated using the Cq values from qRT-PCR of the F1 and F3 in training data (samples originally included in the RNA-seq data) demonstrating a significant relationship between Cqs and behavior (R2=0.70, p=0.01). Lastly, the model generalized to the Cqs of test data, further demonstrating a significant correlation between gene expression and behavior in a novel data set (R2=0.38, p=0.03).
# Conclusions: 
This is the first evidence of transgenerational programming of gene expression and methylation in the PFC via antenatal sGC. Further, we identified 22 genes that are differentially expressed across all 3 generations, the majority of which contain a MEF2C binding site within cis-regulatory distance of the transcription start site. Mef2c expression can be regulated by glucocorticoids. Furthermore, this is the first evidence of a correlation between stress-activated locomotor behavior and gene expression in the PFC following prenatal sGC over multiple generations. Interestingly, this association only occurred in F1 and F3 animals, as F2 animals displayed a distinct expression profile. These findings provide insight into the heritable changes in gene expression affected by prenatal sGC exposure and how these expression changes may relate to behavior. Further, these data have identified key genes for future investigation as drug and intervention targets. 
# Client Recommendations:
1.	 Employ the described analytical pipeline to gene expression and behavioral data from other contexts to elucidate molecular mechanisms involved in those behaviors.
2.	Examine genes identified by this technique for their viability as bio-markers for diseases or exposure
3.	Examine genes identified by this technique for their viability as drug targets

